plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
+ZONE_RETRY = 60 * 60
```

Comment:
What values would you suggest here?  RFC1912 suggests retry should be less than retry; but our retry may be less than usual also.

What about 15 min refresh / 15 min retry?